tests/test_viz.py

```diff
@@ -93,7 +93,7 @@ def test_plot_metadata_alpha_diversity_with_nans(ocx, api_data):
     with mock.patch.object(SampleCollection, "alpha_diversity", return_value=mock_alpha_div_values):
         chart = samples.plot_metadata(vaxis="shannon", return_chart=True)
 
-    assert chart.data["shannon"].tolist() == [5.2120437645419395]
+    assert chart.data["shannon"].tolist() == [5.212043764541939]
```

Comment:
That is example of a bad test. Comparing floats is dangerous. Probably CPU acts slightly differently on ARM versus x86.

```python
shannon_result = chart.data["shannon"].tolist()
assert len(shannon_result) == 1
assert math.isclose(shannon_result[0], 5.212043764541939)
# Or
assert shannon_result[0] == pytest.approx(5.212043764541939)
```

Reply:
Thanks for catching this! Updated to your suggestion.